src/Turbo/README.md

```diff
@@ -74,7 +74,7 @@ The content of a frame can be lazy loaded:
 {% extends 'base.html.twig' %}
 
 {% block body %}
-    <turbo-frame id="the_frame_id" src="{{ path('block') }}">
+    <turbo-frame id="the_frame_id" src="{{ path('block') }}"> // can we make it clearer what "block" is?
```

Comment:
Yes, I'll refactor this to use https://github.com/symfony/symfony/pull/40575 when this will be merged.